exporters/BxDRobotExporter/BxDRobotExporter/BxDRobotExporter/Wizard/Pages/ExportOrAdvancedForm.cs

```diff
@@ -33,5 +33,10 @@ private void exportRobotButton_Click(object sender, EventArgs e)
             Close();
             StandardAddInServer.Instance.ForceExport();
         }
+
+        private void MainLayout_Paint(object sender, PaintEventArgs e)
```

Comment:
Why is this empty?

Reply:
I noticed that too. Those are created as event handlers when double-clicking a component in the WinForm editor. They are often auto-generated by accident and the best way I can think of to remove them is to simply delete the handler and remove it's reference from Designer.cs

Reply:
I can make that change and push again to this branch.